src/components/tz.rs

```diff
@@ -30,13 +29,13 @@ impl TimeZone {
 
 impl TimeZone {
     /// Get the offset for this current `TimeZoneSlot`.
-    pub fn get_offset_nanos_for(&self) -> TemporalResult<BigInt> {
+    pub fn get_offset_nanos_for(&self) -> TemporalResult<i128> {
         // 1. Let timeZone be the this value.
         // 2. Perform ? RequireInternalSlot(timeZone, [[InitializedTemporalTimeZone]]).
         // 3. Set instant to ? ToTemporalInstant(instant).
         // 4. If timeZone.[[OffsetMinutes]] is not empty, return 𝔽(timeZone.[[OffsetMinutes]] × (60 × 10^9)).
         if let Some(offset) = &self.offset {
-            return Ok(BigInt::from(i64::from(*offset) * 60_000_000_000i64));
+            return Ok(i128::from(*offset) * 60_000_000_000);
```

Comment:
We may want to add a comment under the offset property to say its offset minutes from UTC, as it's more descriptively called offsetMinutes in the spec